README.md

```diff
@@ -26,34 +55,57 @@ const rs = await db.execute("SELECT * FROM users");
 console.log(rs);
 ```
 
-Connecting to a remote [libSQL server](https://github.com/libsql/sqld):
+### libSQL sqld instance
+
+To connect to a [libSQL sqld] instance using a [WebSocket URL](#websocket-urls) or [HTTP URL](#http-urls):
```

Comment:
This should perhaps mention that `libsql` implies TLS. This won't work with a sqld instance running locally (or, really, anything else but Turso).

Reply:
The implication with the example below is that access to sqld might involve an auth token - it doesn't matter which transport is in use. I didn't really feel it was necessary to show a whole new example for HTTP when all that changes is the scheme of the URL.

I think what you're saying definitely belongs in the supported URLs section though, which I will definitely add.